x-pack/plugins/monitoring/public/components/table/eui_table_ssp.js

```diff
@@ -97,7 +104,6 @@ export function EuiMonitoringSSPTable({
         items={items}
         pagination={pagination}
         onChange={onChange}
-        loading={isLoading}
```

Comment:
No loading visual was happening in Angular version so I removed it.  Would need to pass this down from parent component at some point.